tools/webpack/shared.js

```diff
@@ -25,7 +25,7 @@ const baseConfig = {
 				parallel: true,
 				terserOptions: {
 					output: {
-						comments: /translators:/i,
+						comments: /(translators:|wp:polyfill)/i,
```

Comment:
Does this need to be changed in `packages/scripts/config/webpack.config.js` as well?

Reply:
I don't think it's necessary unless we want to provide this functionality (automatically marking which files need `wp-polyfill`) to consumers of `@wordpress/scripts`. We would need further changes in order to make that work, though.